raiden-cli/src/utils/channels.ts

```diff
@@ -9,6 +9,25 @@ export function flattenChannelDictionary(channelDict: RaidenChannels): RaidenCha
   );
 }
 
+function transformSdkChannelStateToApi(state: ChannelState): ApiChannelState {
+  let apiState;
+  switch (state) {
+    case ChannelState.open:
+    case ChannelState.closing:
```

Comment:
Kinda.. we don't receive nor send transfers on it, but some action which is only possible on open channels are still possible here, namely closing it (or attempting to do so, in case the previous attempt failed for some recoverable reason). The python API doesn't have these intermediary states, always assume close call will succeed and I think it is some fatal error or some undefined state if it doesn't (we had to be more reliable here due to interactive signing and the possibility of user rejecting the prompt).